src/pages/index.js

```diff
@@ -68,17 +133,60 @@ imageCardPopup.setEventListeners();
 
 //Elements
 
-const userInfo = new UserInfo(".profile__title", ".profile__description");
+const userInfo = new UserInfo(
+  ".profile__title",
+  ".profile__description",
+  ".profile__image"
+);
 
 const editFormValidator = new FormValidator(config, profileEditForm);
 const addFormValidator = new FormValidator(config, addForm);
+const profilePictureValidator = new FormValidator(config, profilePictureForm);
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "e3f5bc64-c279-4474-9c65-8c5ae0831eb9",
+    "Content-Type": "application/json",
+  },
+});
 //Functions
 
 function handleImageClick(cardData) {
   imageCardPopup.open(cardData);
 }
+function handleDeleteCard() {
+  this._cardDeleteButton.addEventListener("click", () => {
+    this._handleDeleteCard();
```

Comment:
Needs correction: the private methods should not be used outside of a class. Please rename tghis method to something like `deleteCard`